querybook/server/lib/query_analysis/validation/validators/base_sqlglot_validator.py

```diff
@@ -39,7 +39,7 @@ def _get_query_validation_result(
         start_index: int,
         end_index: int,
         suggestion: str = None,
-        validation_result_object_type=QueryValidationResultObjectType.SUGGESTION,
+        validation_result_object_type=QueryValidationResultObjectType.LINT,
```

Comment:
why removing the`SUGGESTION` type and using `LINT`?

Reply:
Realized that there's no need to distinguish the two since it requires the same behavior as lint (adding the yellow/red underline to the text and the error message tooltip). If the "suggestion"property is available then we'll display the tooltip to accept the suggestion